### ISSUE TITLE
Daz to blender bridge version error

### ISSUE THREAD

**Issue by Thunderas** [daz3d/DazToBlender#148]
While i install daz to blender bridge for daz studio 4.20, (Daz is showing that it installs bridge version 18.42), in reality when i enable the addon inside blender it shows that the actual version Daz installed is 17.40.   I tried everything but everytime the version that is installed won't be the 18.42 that daz claims.

**Comment by danielbui78** [comment]
Correct.  The Daz Sutdio-based C++ source code was bugfixed and version was incremented but no Blender-based python code was altered, so version remained the same.  The major and minor versions match, only the revision number and build numbers have changed.

To avoid confusion, future version numbering scheme will use the last number as a bugfix and build number, rather than just a build number.  That way, the revision number can stay the same between official releases while being updated with hotfixes, etc.

**Comment by Thunderas** [comment]
Thank you!

**Comment by danielbui78** [comment]
Reopening this issue so that people are aware of the confusion with version numbers that are not actually mismatched.  This confusion in understanding version numbers should be resolved with the next update which will switch to a more intuitive version numbering scheme.
